proposals/0000-kep-process/README.md

```diff
@@ -80,7 +80,7 @@ Some examples include:
 A KEP is broken into sections which can be merged into source control
```

Comment:
Thank you for this @anishasthana!
Should we remove the process from this doc and just keep README file under:
`/proposals/README.md` ?
This README file just says that users should check this KEP template:
https://github.com/kubeflow/community/tree/master/proposals/NNNN-template